packages/databyss-ui/components/Editor/state/reducer.js

```diff
@@ -64,6 +67,74 @@ const setActiveBlockType = (state, type, isNew) => {
   }
 }
 
+const insertBlock = (state, payload) => {
```

Comment:
Let's rename this `insertNewActiveBlock` for clarity and make the args clearer:
`const insertNewActiveBlock = (state, { insertedBlockId, insertedBlockHtml, previousBlockId, previousBlockHtml }) { ... }`

Reply:
i'll do this one